### PR TITLE
Allow users to select color for big number

### PR DESCRIPTION
**Pull request by hughhhh**
Added color picker control to BigNumber Viz

big_number_colors

@kristw @williaster @mistercrunch @betodealmeida

### CODE REVIEW

File: superset/assets/src/modules/colors.js [hunk 1]
```diff
@@ -593,3 +593,12 @@ export function hexToRGB(hex, alpha = 255) {
   const b = parseInt(hex.slice(5, 7), 16);
   return [r, g, b, alpha];
 }
+
+
+export const rgbToHex = function (color) {
+    console.log(color);
```
Comment (mistercrunch) [inline superset/assets/src/modules/colors.js]: oops

File: superset/assets/src/explore/visTypes.jsx [hunk 2]
```diff
@@ -1247,6 +1247,7 @@ export const visTypes = {
           ['compare_lag', 'compare_suffix'],
           ['y_axis_format', null],
           ['show_trend_line', 'start_y_axis_at_zero'],
+          ['color', null],
```
Comment (mistercrunch) [inline superset/assets/src/explore/visTypes.jsx]: I was surprised to see that color isn't `renderTrigger: true`, looks like `color` isn't used anywhere, but `color_picker` is and behaves as expected.

File: superset/assets/src/modules/colors.js [hunk 3]
```diff
@@ -593,3 +593,12 @@ export function hexToRGB(hex, alpha = 255) {
   const b = parseInt(hex.slice(5, 7), 16);
   return [r, g, b, alpha];
 }
+
+
+export const rgbToHex = function (color) {
```
Comment (mistercrunch) [inline superset/assets/src/modules/colors.js]: I think you meant to use `rgbToHex` in `BigNumber.jsx`

File: superset/assets/src/visualizations/BigNumber.jsx [hunk 4]
```diff
@@ -237,6 +237,11 @@ function adaptor(slice, payload) {
   const formatValue = d3FormatPreset(formData.y_axis_format);
   const bigNumber = supportTrendline ? data[data.length - 1][1] : data[0][0];
 
+  const { color } = formData;
+  const userColor = '#' + ('0' + parseInt(color.r, 10).toString(16)).slice(-2) +
```
Comment (mistercrunch) [inline superset/assets/src/visualizations/BigNumber.jsx]: Use the function in `colors.js`

File: superset/assets/src/explore/visTypes.jsx (hunk 2 above)
Comment (mistercrunch) [reply superset/assets/src/explore/visTypes.jsx]: Looks like it has the right default of brandPrimary too.

File: superset/assets/src/modules/colors.js (hunk 1 above)
Comment (kristw) [reply superset/assets/src/modules/colors.js]: Can use `d3-color` instead of writing a new module

**Comment by kristw** [comment]
@hughhhh Code looks good. One question is what is the default color then?

**Comment by mistercrunch** [comment]
@kristw it comes from `color_picker`, which has its default set to brand color, so we're good. https://github.com/apache/incubator-superset/blob/master/superset/assets/src/explore/controls.jsx#L241

**Pull request merged**